### PR TITLE
More gifs

### PR DESCRIPTION
**Pull request by ana0**
Uhhh I added gifs

### CODE REVIEW

File: _includes/2018-hero.html [hunk 1]
```diff
@@ -10,7 +10,7 @@ <h3 style="margin-bottom: 1.3em; margin-top: 0;">Toronto, Ontario</h3>
   <section>
     <div class="feature">
       <a class="splash-link" href="/yami-ichi/"><h4 class="splash-heading">Internet Yami-ichi {% include graphics/bullet.svg %} 13</h4></a>
-      <p>Kickoff with network field trip and Toronto&rsquo;s first Yami-ichi</p>
```
Comment (garrying) [inline _includes/2018-hero.html]: I think we decided on keeping the field trip mention in #60

File: images/yami-ichi/dollarSign.gif [hunk 2]
```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
```
Comment (garrying) [inline images/yami-ichi/dollarSign.gif]: Is this file used on the page? It looks like an html file with a gif extention

**Review by garrying — APPROVED** [review]
Thinking we could add more!
giphy

File: _includes/2018-hero.html (hunk 1 above)
Comment (ana0) [reply _includes/2018-hero.html]: This is a mistake - I took over an old branch.  Fixing

File: images/yami-ichi/dollarSign.gif (hunk 2 above)
Comment (ana0) [reply images/yami-ichi/dollarSign.gif]: Well that's weird af - didn't end up using this one, but didn't realize it wasn't an actual gif.  Removing

**Comment by ana0** [comment]
If anyone finds a doge cursor I think that would be very fitting

**Comment by dcwalk** [comment]
looks great! Can we merge?

**Review by dcwalk — APPROVED** [review]

**Pull request merged**